src/com/google/common/css/compiler/commandline/ClosureCommandLineCompiler.java

```diff
@@ -170,6 +170,10 @@ protected ClosureCommandLineCompiler(JobDescription job,
         "Specify an unrecognized property to whitelist")
     private List<String> allowedUnrecognizedProperties = Lists.newArrayList();
 
+    @Option(name = "--allowed-at-rule", usage =
+      "Specify an @-rule to whitelist, like @supports")
+    private List<String> additionalAtRules = Lists.newArrayList();
```

Comment:
new rules can be specified via `--allowed-at-rule` on the command line, just like `--allowed-unrecognized-property` and other flags